### DISCUSSION TITLE
Interface question for macOS client

### DISCUSSION THREAD

**Discussion by drapiii** [transmission/transmission#5049] [Q&A]
Hello! Why don't I see the "Transmission" label on the Transmission toolbar for macOS like it's shown on the website? I searched everywhere in customize toolbar and other settings but I don't see that option or don't know how to add it.

**Comment by mangosplat** [comment]
It was removed. The screenshots are outdated on the website (it does not receive much maintenance currently apart from releases)

https://github.com/transmission/transmission/pull/3919
